### PR TITLE
Adds icon font

### PR DESCRIPTION
**Pull request by javierarce**
screen shot 2016-01-17 at 22 22 04

### CODE REVIEW

**Comment by javierarce** [comment]
@xavijam, @piensaenpixel: can you review this, please?

I put the files to generate the font in the root of the project, but maybe it makes more sense to have it inside the `src` folder. 

Also, I've added the instructions to generate the font in the readme file.

:*

**Comment by xavijam** [comment]
What do you think if we display all icons in the same "canvas"?

**Comment by javierarce** [comment]
I tried that at first, but there are so many icons that would make very difficult to identify each of them.

**Comment by xavijam** [comment]
Adding them with the name below?

**Comment by javierarce** [comment]
Ok, let me try that.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
+.DS_Store
```
Comment (xavijam) [inline .gitignore:1]: This is not necessary :dancer:

**Comment by javierarce** [comment]
screen shot 2016-01-18 at 11 15 06
screen shot 2016-01-18 at 11 15 12

**Comment by xavijam** [comment]
If you don't like it, we could create a super simple page out of UI styleguide stuff, generated by gulp process (using a template). What do you think?

File: .gitignore (hunk 1 above)
Comment (javierarce) [inline .gitignore:1]: Why not? Doesn't make sense since we are working with macs? Otherwise I always have to remove and keep and eye on them

**Comment by xavijam** [comment]
Or leave it as it is and improve it in the future.

**Comment by javierarce** [comment]
+1 to leave it like now and ask Emilio for a cool design :)

File: .gitignore (hunk 1 above)
Comment (xavijam) [inline .gitignore:1]: Wait... I thought it was a DS_Store file, Seems like I'm still waking up! Sorry buddy :(.

**Comment by xavijam** [comment]
:+1:

File: .gitignore (hunk 1 above)
Comment (javierarce) [inline .gitignore:1]: No problem :)

**Comment by javierarce** [comment]
I've included the generation of the font inside of the base grunt task. It's also possible to run `grunt generate-font` individually.

**Pull request merged**